internal/cmd/skupper/token/kube/token_issue.go

```diff
@@ -80,7 +80,12 @@ func (cmd *CmdTokenIssue) ValidateInput(args []string) error {
 		if !ok {
 			validationErrors = append(validationErrors, fmt.Errorf("there is no active skupper site in this namespace"))
 		} else {
-			cmd.grantName = siteName + "-" + uuid.New().String()
+			ok, siteName = utils.SiteLinkAccessEnabled(siteList)
+			if !ok {
+				validationErrors = append(validationErrors, fmt.Errorf("You must enable link access for this site before you can create a token."))
```

Comment:
I have realized now that while the message is correct, the user must recreate the site to enable the link access; and maybe with this message they could think that the site can be modified.

@pwright do you think that we should say something like "You must have enabled link access for this site before you could create a token."?

Reply:
I agree this is better wording, fixed

Reply:
@nluaces I'm not sure i understand, I've performed `skupper site  update --enable-link-access` as per https://skupperproject.github.io/refdog/commands/site/update.html - are you saying this doesn't always work?

Reply:
@pwright You are completely right, it does always work, thanks for pointing out that.
@lynnemorrison I apologize, the site can be updated and for that reason the wording suggested by Paul is the correct one. Sorry for the confusion.